lib/core/index.js

```diff
@@ -15,6 +15,10 @@ try {
 } catch (err) {} // eslint-disable-line
 
 module.exports = {
+  // Versioned API
+  ServerApiVersion: Object.freeze({
```

Comment:
Do we want this exported from the top level of the driver too?
If so, can we pull it our in to its own variable and attach a small jsdoc comment to it?

Reply:
> Do we want this exported from the top level of the driver too?

Yes, I missed it in the initial back-port.

>  If so, can we pull it our in to its own variable and attach a small jsdoc comment to it?

Good call, will do.